### PR TITLE
feat: rules page

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added rules page

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! And the live preview also looks good and to the point: https://deploy-preview-3--awesome-clarke-46c76b.netlify.app/screens/rules

File: stylesheets/rules.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.rules-container{
+    border: 0.12em solid var(--action-color);
```
Comment (rkapoor10) [inline stylesheets/rules.css]: why ems ?

**Review by rkapoor10 — APPROVED** [review]
The code looks good to merge

File: stylesheets/rules.css (hunk 1 above)
Comment (harshitadaswani) [reply stylesheets/rules.css]: typo! 😅
Thanks for pointing it out. Will change it.

**Pull request merged**